src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/ThunkPool.cs

```diff
@@ -44,8 +45,8 @@ internal static class Constants
         public static readonly int ThunkCodeSize = RuntimeImports.RhpGetThunkSize();
         public static readonly int NumThunksPerBlock = RuntimeImports.RhpGetNumThunksPerBlock();
         public static readonly int NumThunkBlocksPerMapping = RuntimeImports.RhpGetNumThunkBlocksPerMapping();
-        public static readonly uint ThunkBlockSize = (uint)RuntimeImports.RhpGetThunkBlockSize();
-        public static readonly nuint ThunkBlockSizeMask = ThunkBlockSize - 1;
+        public static readonly uint PageSize = Math.Max(BitOperations.RoundUpToPowerOf2((uint)(ThunkCodeSize * NumThunksPerBlock)), BitOperations.RoundUpToPowerOf2((uint)(ThunkDataSize * NumThunksPerBlock + IntPtr.Size)));
```

Comment:
```suggestion
        public static readonly uint PageSize = (uint)BitOperations.RoundUpToPowerOf2(Math.Max(ThunkCodeSize * NumThunksPerBlock, ThunkDataSize * NumThunksPerBlock + IntPtr.Size));
```
Simpler expression?

Reply:
Jupp, was about to change that, but didn't want to interrupt the testing until complete.